### DISCUSSION TITLE
Daten von DBUS unvollständig

### DISCUSSION THREAD

**Discussion by Sven4597** [tbnobody/OpenDTU#1823] [Q&A]
Hallo, ich habe folgendes Problem dass mir die DBUS Daten nicht vollständig angezeigt werden.
Bekomme nur folgende Werte

{"inverters":[{"serial":"138********","name":"Carport","order":0,"data_age":0,"poll_enabled":true,"reachable":true,"producing":true,"limit_relative":100,"limit_absolute":2250}],"total":{"Power":{"v":62.20000076,"u":"W","d":1},"YieldDay":{"v":2593,"u":"Wh","d":0},"YieldTotal":{"v":2.846000195,"u":"kWh","d":3}},"hints":{"time_sync":false,"radio_problem":false,"default_password":false}}

Gruß Sven

**Comment by MetaChuh** [comment]
hi @Sven4597 

danke für die details und screenshots 👍

in der neuen opendtu api findest du deine extra daten unter:
`http://192.168.178.86/api/livedata/status?inv=138290600405`

auf `/api/livedata/status` findet man die basisdaten, sowie die auflistung aller inverter.
details für jeden inverter und allen channels ruft man dann unter `/api/livedata/status?inv=<inverter seriennummer>` ab.

greetings,
metachuh

**Reply by Sven4597** [reply]
Danke für die Info.
Wie mache ich das dann wenn ich die Wechselrichter Daten per dbus an meinen Victron senden möchte. Da kommen keine Daten an.

**Reply by MetaChuh** [reply]
mit dbus kenne ich mich leider garnicht aus.
und die victron wiki:
https://github.com/victronenergy/venus/wiki/dbus
ist für mich auf den ersten blick auch ein spanisches dorf ;-)

bei fragen die rein opendtu spezifisch sind, kann ich eher helfen.
